stingray/crosscorrelation.py

```diff
@@ -1,16 +1,18 @@
 
 import numpy as np
 from scipy import signal
+from scipy.fftpack import ifft
```

Comment:
This should be `scipy.fft` rather than `fftpack`, which is deprecated.

Reply:
Also, note to myself, we should use the same if-statement that tests whether pyfftw is installed here as we do in `Crossspectrum`